aurblobs/constants.py

```diff
@@ -2,10 +2,12 @@
 from xdg.BaseDirectory import xdg_config_home, xdg_cache_home
 
 from . import __PROJECT__ as PROJECT_NAME
+from . import __VERSION__ as PROJECT_VERSION
 
 CONFIG_DIR = os.path.join(xdg_config_home, PROJECT_NAME)
 CACHE_DIR = os.path.join(xdg_cache_home, PROJECT_NAME)
 
 PACMAN_SYNC_CACHE_DIR = os.path.join(CACHE_DIR, 'sync')
 
-DOCKER_IMAGE = 'aurblobs/build'
+DOCKER_IMAGE = 'aurblobs-build:{version}'.format(version=PROJECT_VERSION)
```

Comment:
Why not use "aurblobs/build"? This way we would stick to the default Docker namingconvention also used on DockerHub. If we stick to "aurblobs-build" we can't release our images over DH. Even if we won't do this, I would stick to the default naming scheme.